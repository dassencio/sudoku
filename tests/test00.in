2 2
1 _ _ 2
2 _ 4 _
_ 2 1 4
4 _ _ 2


