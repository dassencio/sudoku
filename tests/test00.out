Sudoku puzzle is not valid.
